src/rand.rs

```diff
@@ -63,18 +63,17 @@ impl Rng {
         Self::default()
     }
 
-    #[cfg_attr(feature = "cargo-clippy", allow(should_implement_trait))]
-    pub fn next(&mut self) -> u32 {
+    pub fn gen_int(&mut self) -> u32 {
         self.pcg32()
     }
 
-    pub fn next_in_range(&mut self, min: u32, max: u32) -> u32 {
+    pub fn gen_int_interval(&mut self, min: u32, max: u32) -> u32 {
         (self.pcg32() % (max - min)) + min
     }
 
-    pub fn next_double(&mut self) -> f64 {
+    pub fn gen_double_interval_unit(&mut self) -> f64 {
         let max = f64::from(u32::MAX);
```

Comment:
Why `interval_unit`?

Reply:
Because it's between `0.0` and `1.0`.
That's called the [unit interval](https://en.wikipedia.org/wiki/Unit_interval).

Reply:
Hmmm, given the meaning of `unit` in Rust, I like Remi's suggestion of `gen_double_interval_01`.

Reply:
I did not know that the 0-1 interval had a name so I think it's pretty fitting. That being said a bit of doc could alleviate the initial surprise here.

I'm fine with either variant though.